packages/relay-runtime/store/observeFragmentExperimental.js

```diff
@@ -173,6 +170,67 @@ function observeSelector<TFragmentType: FragmentType, TData>(
   });
 }
 
+function observePluralSelector<
+  TFragmentType: FragmentType,
+  TData: Array<mixed>,
+>(
+  environment: IEnvironment,
+  fragmentNode: Fragment<TFragmentType, TData>,
+  fragmentSelector: PluralReaderSelector,
+): Observable<FragmentState<TData>> {
+  const snapshots = fragmentSelector.selectors.map(selector =>
+    environment.lookup(selector),
+  );
+
+  return Observable.create(sink => {
+    const states = snapshots.map((snapshot, index) =>
+      snapshotToFragmentState(
+        environment,
+        fragmentNode,
+        fragmentSelector.selectors[0].owner,
+        snapshot,
+      ),
+    );
+
+    const reduceState = () =>
+      states.reduce<FragmentState<TData>>(
+        (acc, state) => {
+          if (acc.state !== 'ok') {
+            return acc;
+          }
+
+          if (state.state === 'ok') {
+            acc.value.push(state.value);
+            return acc;
+          } else {
+            return state;
+          }
+        },
+        {
+          state: 'ok',
+          // $FlowFixMe[incompatible-cast]: TData is enforced to be an array
```

Comment:
Not sure why this should be marked as `$FlowFixMe` when `TData` has `: Array<mixed>` enforcement